Tests/eppo/ClientTests.swift

```diff
@@ -172,7 +172,17 @@ final class eppoClientTests: XCTestCase {
                     XCTFail("Unknown value type: \(testCase.valueType)");
             }
             
+
             XCTAssertTrue(loggerSpy.wasCalled, "Assignment logger was not called.")
+            if let lastAssignment = loggerSpy.lastAssignment {
+                XCTAssertTrue(lastAssignment.experiment.contains(lastAssignment.allocation))
+                XCTAssertTrue(lastAssignment.experiment.count > lastAssignment.allocation.count)
+
+                XCTAssertTrue(lastAssignment.experiment.contains(lastAssignment.featureFlag))
+                XCTAssertTrue(lastAssignment.experiment.count > lastAssignment.featureFlag.count)
+            } else {
```

Comment:
as the underlying assignment expectations are not readily available; I am using these assertions as a proxy for the expected behavior.